### PR TITLE
Additional e2e tests, and some conveniences

### PR DESCRIPTION
**Pull request by johnbelamaric**
- Adds a launch script for use in building / testing e2e tests on GCE
- Adds `k` alias
- Adds test for deploying operators
- Adds regional labels to regional cluster in the automation

### CODE REVIEW

File: e2e/tests/005-regional-free5gc-smf.yaml [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
     repo: free5gc-packages
-    package: free5gc-smf
-    revision: v1
-  downstream:
-    repo: regional
-  annotations:
-    approval.nephio.org/policy: initial
-  injectors:
-  - name: regional
+    package: pkg-example-amf-bp
```
Comment (debeaueric) [inline e2e/tests/005-regional-free5gc-smf.yaml:8]: Should be pkg-example-smf-bp

Comment (johnbelamaric) [reply e2e/tests/005-regional-free5gc-smf.yaml:8]: ooo, thanks!

**Comment by johnbelamaric** [comment]
Ok, this now covers, in automated e2e tests, the README all the way through to step 11.

File: e2e/launch.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+BRANCH=${4:-main}
+
+STARTUP_SCRIPT_URL="https://raw.githubusercontent.com/$REPO/$BRANCH/e2e/provision/gce_init.sh"
+FULLREPO="https://github.com/$REPO.git"
+
+gcloud compute instances delete -q "$VM" || echo
+gcloud compute instances create   \
+  --machine-type e2-standard-8    \
+  --boot-disk-size 200GB          \
+  --image-family=ubuntu-2204-lts  \
```
Comment (electrocucaracha) [inline e2e/launch.sh:31]:
```suggestion
  --image-family=ubuntu-2004-lts  \
```

File: e2e/provision/gce_init.sh [hunk 3]
```diff
@@ -45,6 +52,8 @@ cd ./test-infra/e2e/provision
 export DEBUG DEPLOYMENT_TYPE
 runuser -u ubuntu ./gce_install_sandbox.sh
 
+usermod ubuntu -G docker -a
```
Comment (electrocucaracha) [inline e2e/provision/gce_init.sh:55]: I was planning to submit a PR about this, but given that you already have it

```suggestion
# Grant Docker permissions to current user
if ! getent group docker | grep -q "$USER"; then
    sudo usermod -aG docker "$USER"
fi
```

**Comment by electrocucaracha** [comment]
@johnbelamaric I think that eventually we can add multi-cloud support to the terraform scripts and use them instead of the `launch.sh` bash script, right?

**Comment by johnbelamaric** [comment]
Yeah, the launch.sh is just for me to do my testing but I figured I would share it.

**Pull request merged**